open_spiel/python/egt/alpharank_visualizer.py

```diff
@@ -356,7 +398,13 @@ def plot_pi_vs_alpha(pi_list,
   rank = 1
   num_strats_printed = 0
   add_legend_entries = True
-  for mass, strats in sorted(masses_to_strats.items(), reverse=True):
+
+  sorted_masses_strats = generate_sorted_masses_strats(
```

Comment:
Done. I've also added a similar flag to sweep_pi_vs_alpha in alpharank.py (default False, such that your old plots are the defaults), because that's the function that most people would usually call (rather than directly calling the plot function).